src/log/log.c

```diff
@@ -2045,7 +2045,7 @@ int
 __wt_log_scan(WT_SESSION_IMPL *session, WT_LSN *lsnp, uint32_t flags,
```

Comment:
`*lsnp` should be renamed to `*start_lsnp` and the new one should be added as the 3rd arg to be `*end_lsnp`.